docs/1.guide/2.routing.md

```diff
@@ -27,6 +27,20 @@ routes/
 nitro.config.ts
 ```
 
+You can nest routes by creating subdirectories.
+
+```md
+routes/
+  communities/
+    index.get.ts
+    index.post.ts
+    [id]/
+        index.get.ts
+        index.post.ts
+  hello.get.ts
+  hello.post.ts
```

Comment:
```suggestion
server/
  routes/
    communities/
      index.get.ts
      index.post.ts
      [id]/
          index.get.ts
          index.post.ts
    hello.get.ts
    hello.post.ts
```

Reply:
I think it would be nice to update all examples to include `server/` for consistency.